README.md

```diff
@@ -321,7 +321,22 @@ For information on the transition triggers, refer to:
 * description of the `timeout`, `ttl timeout`, and `ttr timeout` options in
 the sections of the corresponding [queue types](#queue-types).
 
-![Task state diagram](./doc/images/statediagram.svg)
+```mermaid
+flowchart LR
+      I(("init"))-->  |"put()"| R[READY]
+      I(("init"))--> |"put('my_task_data',{ delay = delay })"| DE[DELAYED]
+      R[READY]--> |"take()"| T[TAKEN]
```

Comment:
BTW, can't we use just `READY`, `TAKEN`, `DONE` and so on instead of repeating `R[READY]`, `T[TAKEN]`, `DO[DONE]`?